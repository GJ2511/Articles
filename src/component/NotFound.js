import React from 'react';
import { Link } from 'react-router-dom';

function NotFound() {
    return (
        <div className="container">
            <div className="row text-center mt-5">
                <div className="col-md-12">
                    <div className="error-template">
                        <h1>Oops!</h1>
                        <h2>404 Not Found</h2>
                        <div className="error-details">Sorry, an error has occured, Requested page not found!</div>
                        <div className="error-actions mt-2">
                            <Link to="/" className="btn btn-primary btn-lg rounded-pill">
                                <span className="glyphicon glyphicon-home"></span>
                                Take Me Home{' '}
                            </Link>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
}

export default NotFound;
