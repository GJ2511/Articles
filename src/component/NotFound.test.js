import React from 'react';

import NotFound from './NotFound';

describe('<NotFound tests />', () => {
    test('match snapshot', () => {
        expect(shallow(<NotFound />)).toMatchSnapshot();
    });
});
