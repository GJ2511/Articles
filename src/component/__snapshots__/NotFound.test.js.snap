// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound tests /> match snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row text-center mt-5"
  >
    <div
      className="col-md-12"
    >
      <div
        className="error-template"
      >
        <h1>
          Oops!
        </h1>
        <h2>
          404 Not Found
        </h2>
        <div
          className="error-details"
        >
          Sorry, an error has occured, Requested page not found!
        </div>
        <div
          className="error-actions mt-2"
        >
          <Link
            className="btn btn-primary btn-lg rounded-pill"
            to="/"
          >
            <span
              className="glyphicon glyphicon-home"
            />
            Take Me Home
             
          </Link>
        </div>
      </div>
    </div>
  </div>
</div>
`;
